utils/mstpctl-utils-functions.sh

```diff
@@ -2,7 +2,7 @@ mstpctl_parse_ports()
 {
   while [ x"${1+set}" = xset ]
   do
-    # For compatibility: the `all' option.
+    # For compatibility: the 'all' option.
```

Comment:
utils/mstpctl-utils-function.sh is copied almost verbatim from /lib/bridge-utils/bridge-utils.sh in the Debian bridge-utils package.  These changes look good to me, but just be aware that applying these changes means we are forking the development of this file.